features/foutafhandeling.feature

```diff
@@ -0,0 +1,135 @@
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((a b,c d))                                                                                       | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((134647.123))                                                                                    | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((134647.123 457842.456, 134747.789 d))                                                           | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON (([meer dan 350 coördinaten]))                                                                    | geometrie.intersects | maxItems         | Array bevat meer dan {maxItems} items.   |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((5.96072575 52.18450437,5.95270058 52.17811002, 5.96793552 52.17816266, 5.96072575 52.18450437)) | geometrie.intersects | geometryMismatch | Waarde is niet conform opgegeven CRS.    |
+
+    Abstract Scenario: waarde van AdresBody.geometrie.intersects parameter overschrijdt maximale toegestane oppervlakte 
+        Als /adressen met request body AdresBody.geometrie.intersects=POLYGON ((134647 457842, 137512 457842, 137512 455907, 134647 455907, 134647 457842)) wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name                           | code           | reason                                             |
+        | AdresBody.geometrie.intersects | surfaceMaximum | Waarde is hoger dan maximum oppervlakte 250000 m2. |
+
+Rule: verplichte request body is opgegeven
+
+    Scenario: verplichte request body is niet opgegeven
+        Als /adressen zonder request body AdresBody wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                               |
+        | title  | Geef tenminste één parameter op.     |
+        | status | 400                                  |
+        | code   | paramsRequired                       |
+        | detail | Request body moet worden meegegeven. |
+        En bevat de response geen invalidParams
+
+Rule: verplichte request body parameter is opgegeven
```

Comment:
dit is herhaling van de rule op regel 100

Reply:
Aangepast